crates/ruma-events/tests/it/sticky.rs

```diff
@@ -0,0 +1,129 @@
+use assert_matches2::assert_matches;
+use js_int::uint;
+use ruma_common::{serde::CanBeEmpty, MilliSecondsSinceUnixEpoch};
+use ruma_events::{AnyMessageLikeEvent, MessageLikeEvent, StickyDurationMs, StickyObject};
+use serde_json::{from_value as from_json_value, json};
+
+#[test]
+fn new_wrapping_keeps_in_range_values() {
```

Comment:
```suggestion
fn new_clamped_keeps_in_range_values() {
```

And same for other tests below.